GoogleForCreators/web-stories-wp
# Enable new `AmpStoryCssOptimizer` transformer

## Context



`AmpStoryCssOptimizer` is a new performance-related transformer available in the AMP toolbox.

## Summary



Enables the new `AmpStoryCssOptimizer` transformer for stories.

## Relevant Technical Choices



Updates Composer dependencies

Also updates PHPStan

## To-do



None

## User-facing changes



None

## Testing Instructions






This PR can be tested by following these steps:

1. Create a new story and preview it on the frontend
2. See `https://cdn.ampproject.org/v0/amp-story-1.0.css` file being loaded
3. See `<script amp-story-dvh-polyfill="">` inline style


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #10154
